Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -920,7 +920,7 @@ Sollten Sie den Test in der App gelöscht haben, können Sie ihn aus dem Papierk
     <!-- YTXT: instruction text for QR code scanning  -->
     <string name="qr_code_scan_body_title">"Welche QR-Codes können Sie scannen?"</string>
     <!-- YTXT: instruction text for QR code scanning  -->
-    <string name="qr_code_scan_body_subtitle">"Ihre PCR-Tests, Schnelltests, Testzertifikate, Impfzertifikate, Genesenenzertifikate sowie Check-ins."</string>
+    <string name="qr_code_scan_body_subtitle">"Tests, Zertifikate, Check-ins und Nachweise bei Ticketbuchungen"</string>
```

Comment:
Can we please still leave the "Ihre Zertifikate"? There are more and more reports that the UQS in the CWA is used to _verify_ certificates from others.

Reply:
Thanks, yes, I'll add that.

Reply:
Thanks @KathrinLuecke!